src/v2/Apps/Artist/Routes/AuctionResults/Components/AuctionResultsControls.tsx

```diff
@@ -23,9 +24,12 @@ export const AuctionResultsControls = ({ artist, toggleMobileActionSheet }) => {
                 Filter
               </Flex>
             </Button>
-            <SortSelect />
+            <Flex ml={1}>
+              <SortSelect />
+            </Flex>
```

Comment:
**Note:** Added some margin here to create some space between the select and the filter button on small screens.